### DISCUSSION TITLE
Grafana Value Mapping

### DISCUSSION THREAD

**Discussion by Snoopynerd** [grafana/grafana#56569] [General]
I have a query A:
SELECT distinct("value") FROM "HMI_O.[1]" WHERE ("Controller Name" = 'Controller1') AND $timeFilter GROUP BY time($__interval) fill(null)

The PLC tag  HMI_O.[1] = displays a number ranging from 1-10

How do I map each values to a certain text :

For e,g
- If the tag HMI_O.[1] outputs value 1 , i want the dashboard to display Alpha

- If the tag HMI_O.[1] outputs value 2 , i want the dashboard to display Beta
- If the tag HMI_O.[1] outputs value 3 , i want the dashboard to display Gamma 
- - If the tag HMI_O.[1] outputs value 4 , i want the dashboard to display Delta 
and so on

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
